packages/strapi-admin/admin/src/components/Users/LinkNotification/index.js

```diff
@@ -0,0 +1,47 @@
+// This component is a work in progress
+// It's made to be used when the users API is ready
+import React from 'react';
+import { Flex, Text } from '@buffetjs/core';
+import { Duplicate } from '@buffetjs/icons';
+import PropTypes from 'prop-types';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+import IconWrapper from './IconWrapper';
+import Envelope from './Envelope';
+import Wrapper from './Wrapper';
+
+const LinkNotification = ({ link, description }) => {
```

Comment:
I'm feeling that `<Link link />` is a bit redundant. WYT of using `<Link target />` or `<Link path />` instead?

Reply:
Of course, it is much better :)